package.json

```diff
@@ -49,7 +49,13 @@
     "karma-remap-istanbul": "0.6.0",
     "karma-sourcemap-loader": "0.3.7",
     "karma-typescript-preprocessor": "0.3.1",
-    "powerbi-visuals-tools": "1.13.1",
+    "lodash": "4.16.2",
+    "powerbi-models": "^1.0.2",
+    "powerbi-visuals-tools": "1.13.0",
+    "powerbi-visuals-utils-chartutils": "1.5.1",
+    "powerbi-visuals-utils-dataviewutils": "1.2.0",
+    "powerbi-visuals-utils-formattingutils": "3.0.1",
+    "powerbi-visuals-utils-interactivityutils": "3.1.0",
```

Comment:
These added dependencies are already in `dependencies`. Please remove useless from `devDependencies`